slippage_config.json

```diff
@@ -36,10 +36,10 @@
             }
         },
         {
-            "name": "pump_new_graduated",
+            "name": "pump_new_graduate",
             "range": {
                 "min": 500,
-                "max": 1000
+                "max": 2000
```

Comment:
We have been using the max from `degen` as a temporary substitute. Changing this max to 2000bps for new graduate so that we can revert to using this category correctly.